src/components/UserInfo.js

```diff
@@ -0,0 +1,31 @@
+export default class UserInfo {
+  constructor({ titleSelector, descriptionSelector, avatarSelector }) {
+    this._title = document.querySelector(titleSelector);
+    this._description = document.querySelector(descriptionSelector);
+    this._avatar = document.querySelector(avatarSelector);
+  }
+
+  //new phrasing below 6.5
+  getUserInfo() {
+    return {
+      title: this._title.textContent,
+      description: this._description.textContent,
+    };
+  }
+
+  setUserInfo({ title, description }) {
+    this._title.textContent = title;
+    this._description.textContent = description;
+  } //split the sections 6.9
+
+  setAvatar({ avatar }) {
+    this._avatar.src = avatar;
+  } //split the sections 6.9
+
+  getAvatar({ avatar }) {
```

Comment:
Needs correction: Getters are the functions, which get values (they should return something, not set)